prody/proteins/header.py

```diff
@@ -1041,7 +1041,11 @@ def buildBiomolecules(header, atoms, biomol=None):
 
     biomt = header.get('biomoltrans')
     if not isinstance(biomt, dict) or len(biomt) == 0:
-        raise ValueError("header doesn't contain biomolecular transformations")
+        if header.get('experiment') != 'X-RAY DIFFRACTION':
```

Comment:
Is this robust? Does every PDB file have an 'experiment' entry that indicates its type? And is the entry strictly spelt as 'experiment'?

Reply:
I think so

Reply:
I've now replaced it with a warning always with returning of the original structure. We may want to return None instead. I'm not sure. It may also be something to handle in parsePDB when we provide multiple PDB IDs and only some of them have headers with biomoltrans. Perhaps returning None and then looking for None is a good way to go.

Reply:
I've now pushed here that change too.